dm_cli/import_entity.py

```diff
@@ -0,0 +1,154 @@
+import io
+import json
+from json import JSONDecodeError
+from pathlib import Path
+from typing import Dict, List, Union
+from uuid import uuid4
+
+from .dmss import dmss_api
+from .dmss_api.exceptions import ApiException
+from .domain import Dependency
+from .enums import PRIMITIVES, SIMOS
+from .utils import concat_dependencies, document_already_exists, resolve_reference
+
+keys_to_check = ("type", "attributeType", "extends", "_blueprintPath_")  # These keys may contain a reference
+
+
+def replace_relative_references(
```

Comment:
alot of "duplicated" code here  :/ 
I can have a closer look at it if you think there is room for improvement

Reply:
Refactored and merged into a single, shared function!